### PR TITLE
Issue #9142: resolve Assertions.assertFalse

### PR DESCRIPTION
**Pull request by rnveach**
Issue #9142

### CODE REVIEW

File: src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java [hunk 1]
```diff
@@ -534,11 +533,14 @@ public void testAllCheckstyleChecksHaveMessage() throws Exception {
...
-                        .that(messages).isEmpty();
+                        .that(messages)
+                        .isEmpty();
             }
             else {
-                assertFalse(messages.isEmpty(),
-                        name + " should have at least one 'MSG_*' field for error messages");
+                assertWithMessage(
+                        name + " should have at least one 'MSG_*' field for error messages")
+                                .that(messages.isEmpty())
```
Comment (pbludov) [inline src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java:542]: Better to use `isNotEmpty()`

**Review by pbludov — CHANGES_REQUESTED** [review]
Please change all `.isEmpty()).isFalse()` to `.isNotEmpty()`

File: src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java (hunk 1 above)
Comment (rnveach) [reply src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java:542]: Done.

**Review by pbludov — APPROVED** [review]

File: src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionXpathSingleFilterTest.java [hunk 2]
```diff
@@ -138,8 +140,7 @@ public void testIncorrectQuery() {
         }
         catch (IllegalArgumentException ex) {
             assertWithMessage("Message should be: Unexpected xpath query")
-                    .that(ex.getMessage().contains("Incorrect xpath query"))
-                    .isTrue();
+                    .that(ex.getMessage().contains("Incorrect xpath query")).isTrue();
```
Comment (romani) [inline src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionXpathSingleFilterTest.java:143]: Contains can be part of chain call

File: src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionXpathSingleFilterTest.java [hunk 3]
```diff
@@ -188,8 +189,7 @@ public void testInvalidFileRegexp() {
         }
         catch (PatternSyntaxException ex) {
             assertWithMessage("Message should be: Unclosed character class")
-                    .that(ex.getMessage().contains("Unclosed character class"))
-                    .isTrue();
+                    .that(ex.getMessage().contains("Unclosed character class")).isTrue();
```
Comment (romani) [inline src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionXpathSingleFilterTest.java:192]: Contains can be part of chain call

**Review by romani — APPROVED** [review]
Approved With items:

File: src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionXpathSingleFilterTest.java (hunk 2 above)
Comment (rnveach) [reply src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionXpathSingleFilterTest.java]: Done.

**Pull request merged**